### PR TITLE
sync-latest-fx-prices

### LINKED ISSUE #565: SOR: Handle failing FX pools by adding latestFXPrice to it's poolTokens.
- [x] Find out if we need FX pools. - we do!
- [x] FX pool tokens are missing latestFXPrice - subgraph has it and it needs to be added to the DB as well.
- [x] Once done uncomment FX pool from the query filter: https://github.com/beethovenxfi/beethovenx-backend/blob/v3-canary/modules/sor/sorV2/sorV2.service.ts/#L361

### PR DESCRIPTION
**Pull request by gmbronco**
Adding a job for syncing latest FX price from subgraph. The task itself is fast, so I'm scheduling it as one job for all the chains.

closes #565

### CODE REVIEW

**Comment by franzns** [comment]
As discussed, I would refrain from introducing the "multichain-schedule" for this. Would like a more generalized approach.

**Comment by gmbronco** [comment]
Removed multichain job in favour of chain specific ones.

File: modules/network/optimism.ts [hunk 1]
```diff
@@ -406,5 +409,9 @@ export const optimismNetworkConfig: NetworkConfig = {
             name: 'feed-data-to-datastudio',
             interval: (env.DEPLOYMENT_ENV as DeploymentEnv) === 'canary' ? every(5, 'minutes') : every(1, 'minutes'),
         },
+        {
+            name: 'sync-latest-fx-prices',
+            interval: every(10, 'minutes'),
+        },
```
Comment (franzns) [inline modules/network/optimism.ts]: we dont have fx pools on OP. not sure about the other chains

File: modules/network/zkevm.ts [hunk 2]
```diff
@@ -292,5 +292,9 @@ export const zkevmNetworkConfig: NetworkConfig = {
             name: 'feed-data-to-datastudio',
             interval: (env.DEPLOYMENT_ENV as DeploymentEnv) === 'canary' ? every(5, 'minutes') : every(1, 'minutes'),
         },
+        {
+            name: 'sync-latest-fx-prices',
+            interval: every(10, 'minutes'),
+        },
```
Comment (franzns) [inline modules/network/zkevm.ts]: same, think they dont have fx pools

Comment (gmbronco) [reply modules/network/zkevm.ts]: true, FX is on: Mainnet, Polygon and Avalanche so far.

**Pull request merged**